module/observationSubmissions/helper.js

```diff
@@ -55,14 +55,13 @@ module.exports = class ObservationSubmissionsHelper {
           });
         }
 
-        let submissionDocuments;
-
-        if (sortedData !== 'all') {
+        let submissionDocuments; 
+        if (sortedData !== 'all') { 
```

Comment:
@borkarsaish65 why we  are directly calling models from helper? this should happen from query file

Reply:
This is the way it has been already I didnt change or added this new function. I will note this down and make changes later as it will delay my deadline.